.ci/acceptance-tests/inspec-integration.sh

```diff
@@ -17,16 +17,27 @@ export GOPATH=${PWD}/go
 # to disk for use in tests.
 echo "${TERRAFORM_KEY}" > /tmp/google-account.json
 
-git clone https://github.com/slevenick/inspec-gcp.git
+export CLOUD_SDK_REPO="cloud-sdk-stretch"
+echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+apt-get update && apt-get install google-cloud-sdk -y
 
-# new train plugin not published yet, install locally for now
-pushd inspec-gcp
-bundle
+gcloud auth activate-service-account terraform@graphite-test-sam-chef.iam.gserviceaccount.com --key-file=$GOOGLE_CLOUD_KEYFILE_JSON
```

Comment:
Do we have to use gcloud to get the service account working? Seems like a lot of overhead

Reply:
gcloud/gsutils is using the provided service account to pull down the contents of a private bucket that contain my train plugin until it is published. Planning on also storing VCR cassettes there